DependencyInjection/Configuration.php

```diff
@@ -576,6 +576,7 @@ private function getOrmEntityManagersNode(): ArrayNodeDefinition
                     ->scalarNode('quote_strategy')->defaultValue('doctrine.orm.quote_strategy.default')->end()
                     ->scalarNode('entity_listener_resolver')->defaultNull()->end()
                     ->scalarNode('repository_factory')->defaultValue('doctrine.orm.container_repository_factory')->end()
+                    ->scalarNode('enable_metadata_cache')->defaultValue(!$this->debug)->end()
```

Comment:
From reading the code, I see that by default, there still is a metadata cache, it's just that its duration is limited by the duration of the current execution of PHP.
```suggestion
                    ->scalarNode('enable_persistent_metadata_cache')->defaultValue(!$this->debug)->end()
```

Reply:
Technically you are right. 

But isn't that the case with all the things that are loaded? They are there during the lifecycle of the request / while the application runs.

For an end user, you want to cache (persist) it or not.